### ISSUE TITLE
Toggle permanent sprint allows weird behavior

### ISSUE THREAD

**Issue by UltraBlackLinux** [maruohon/tweakeroo#214]
Hey there,
I just noticed, that the autosprint tweak allows the user to sprint in 1b high water while standing - this doesn't seem to be a vanilla feature
Please fix that,
Thanks!

**Comment by barraIhsan** [comment]
Yes, I also already mention it in [discord](https://discord.com/channels/211786369951989762/453662800460644354/775319443898236928)
